opm/simulators/timestepping/AdaptiveTimeStepping.hpp

```diff
@@ -426,24 +426,37 @@ void registerAdaptiveParameters();
                             // Found no wells to close, chop the timestep as above.
                             chopTimestep();
                         } else {
-                            // Close all consistently failing wells.
+                            // Close all consistently failing wells that are not under group control
                             int num_shut_wells = 0;
```

Comment:
If I did not read it wrong, since you have a `shut_wells` keep tracking of things, `num_shut_wells` is not needed anymore.